src/cross.js

```diff
@@ -164,7 +164,7 @@ const asResultAccount = (unsafe: mixed): AccountData => {
     index,
     balance,
     // $FlowFixMe No idea how to make flow like this
```

Comment:
Can you remove this flowfixme then? I imagine it was why it was complaining

Reply:
I'll check